presto-product-tests/src/main/resources/sql-tests/testcases/hive_tpch/q1.sql

```diff
@@ -1,4 +1,4 @@
--- database: presto; groups: tpch, big_query; tables: lineitem
```

Comment:
Is this commit independent from adding support for correlated scalar subqueries? If so, I can merge it while you iterate on the other commits.